docs/UserGuide.md

```diff
@@ -404,11 +404,91 @@ Examples:
 * `importcsv n/2`
   * Reads from CSV treating the 2nd row as names, 3rd row as phone numbers, 4th row as email address, 5th row as tagged.
 
-
 ### Archiving data files `[coming in v2.0]`
 
 _Details coming soon ..._
 
+###Extension on Find contact: `find` `t/`
+
+Locates persons by different search types
+
+Format: `t/SEARCH_TYPE k/KEYWORD [MORE_KEYWORDS]`
+
+* The search is case-insensitive. e.g. hans will match Hans
+* The order of the keywords does not matter. e.g. Hans Bo will match Bo Hans
+* `SEARCH_TYPE` must match one of the following:
+  * Name
+  * Address
+  * Phone
+  * Email
+* Persons matching at least one keyword will be returned
+
+Examples:
+* `find t/Address k/Garden`
+    * Lists all contact with address contains keyword “Garden”
+* `find t/Phone k/87438807`
+    * Lists all contact with phone matches keyword “87438807”
+
+### Batch and Range Commands
+
+### Fetch contacts by Condition Clause: `where/`
+
+If the given condition is satisfied, it returns a group of contacts
+
+Format: `batch COMMAND where/CONDITION`
+
+* The commands include list, edit, delete
```

Comment:
I suggest using a stricter language because the allowed commands are exhaustive.
```suggestion
* The allowed operations in `COMMAND` are
  * list
  * edit
  * delete
```